MobileWeather/MainViewController.m

```diff
@@ -48,6 +48,8 @@ - (void)viewDidLoad {
     
     self.versionLabel.text = [NSString stringWithFormat:@"%@ (%@)", bundleAppVersion, bundleBuildVersion];
 
+    self.conditionIcon.contentMode = UIViewContentModeScaleAspectFill;
```

Comment:
You can change this in the storyboard by clicking on the image view, then changing "Content Mode" here:
![Screen Shot 2021-07-28 at 8 57 42 AM](https://user-images.githubusercontent.com/1767063/127325889-646ed737-4d7e-4a90-a15c-4ce6815d6554.png)